src/app/business/delfin/storage-details/storage-details.component.ts

```diff
@@ -453,8 +453,46 @@ export class StorageDetailsComponent implements OnInit {
     }
 
     getAllActiveAlerts(storageId){
-        this.ds.getAlertsByStorageId(storageId).subscribe((res)=>{
-            this.allActiveAlerts = res.json().alerts;
+        this.ds.getAllAlerts().subscribe((res)=>{
+            let alertsFromAlertManager = res.json().data;
+            alertsFromAlertManager.forEach(element => {
+                if(element.labels.storage_id == storageId){
+                    let alert = {
```

Comment:
Is it possible to keep this as part of some Constants? Advantage: in future if we add new attribs and want to use at different places then its easier

Reply:
This is fixed. Moved the alert data model to the consts file.